webpack.config.search.js

```diff
@@ -38,11 +38,7 @@ module.exports = [
 			filename: 'jp-search.bundle.js',
 		},
 		performance: isDevelopment
-			? {
-					maxAssetSize: 500000,
-					maxEntrypointSize: 500000,
-					hints: 'error',
-			  }
+			? false
```

Comment:
Makes sense 👍 

What do we lose (if anything) by omitting `hints: 'error'`?

Reply:
Omitting the `hints` value in the production `performance` config would disable the asset size check. [According to the docs](https://webpack.js.org/configuration/performance/#performancehints), setting it to `error` is considered best practice for production bundles.